WordPress/UITestsFoundation/Globals.swift

```diff
@@ -36,6 +36,15 @@ public func pullToRefresh(app: XCUIApplication = XCUIApplication()) {
     top.press(forDuration: 0.01, thenDragTo: bottom)
 }
 
+public func waitForExistAndTap(_ element: XCUIElement, timeout: TimeInterval = 5) {
+    guard element.waitForExistence(timeout: timeout) else {
+        XCTFail("Expected element (\(element)) did not exist after \(timeout) seconds.")
```

Comment:
I think present tense fits better here:

```suggestion
        XCTFail("Expected element (\(element)) does not exist after \(timeout) seconds.")
```